spec/index.html

```diff
@@ -5445,9 +5444,7 @@ <h3>Operator Mapping</h3>
               <td><span class="type RDFterm"><a data-cite="RDF12-CONCEPTS#dfn-triple-term">Triple Term</a></span></td>
               <td><span class="type RDFterm"><a data-cite="RDF12-CONCEPTS#dfn-triple-term">Triple Term</a></span></td>
               <td>
-                ( A.subject != B.subject ) <a href="#logical-or-operator" class="SPARQLoperator">||</a><br/>
-                ( A.predicate != B.predicate ) <a href="#logical-or-operator" class="SPARQLoperator">||</a><br/>
-                ( A.object != B.object )
+                <a data-cite="XPATH-FUNCTIONS-31#func-not">fn:not</a>(<a data-cite="XPATH-FUNCTIONS-31#func-numeric-equal">op:numeric-equal</a>(<a href="#func-compareTripleTerm">compareTripleTerm</a>(A, B), 0))
```

Comment:
Making source more readable. A little manual `tidy`.
```suggestion
                <a data-cite="XPATH-FUNCTIONS-31#func-not">fn:not</a>(<a
                data-cite="XPATH-FUNCTIONS-31#func-numeric-equal">op:numeric-equal</a>(<a
                href="#func-compareTripleTerm">compareTripleTerm</a>(A, B), 0))
```